### DISCUSSION TITLE
Is the PorousFlowEffectiveStressCoupling kernel compulsory?

### DISCUSSION THREAD

**Discussion by steam-simulator** [idaholab/moose#26020] [Q&A Modules: Porous Flow] [answered]
Hello,
I’m conducting a Thermo-hydro-mechanical simulation using the porous flow module. I’m wondering whether the `PorousFlowEffectiveStressCoupling` kernel is compulsory for this simulation. When is it advisable to ignore it? Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

If you remove this you are decoupling the thermo-hydro and the mechanical parts. The fluid pressure gradient stress term wont exist to affect the displacements. 

Guillaume
